components/add_new_cv_form.tsx

```diff
@@ -97,26 +86,33 @@ export default function AddNewCvForm({ id }: Props) {
     }
 
     delete updatedCv.projects;
+    delete updatedCv.technical_skills;
 
     return supabase.from("cv").upsert(updatedCv).select();
   }
 
   async function upsertProjects(values: any, cvData: any) {
-    if (values.id) {
-      return supabase.from("projects").update(values).eq("id", values.id);
-    }
-    return supabase.from("projects").insert({ ...values, cv_id: cvData[0].id });
+    values.cv_id = cvData[0].id;
+    return supabase.from("projects").upsert(values);
```

Comment:
So this function is no longer used in this PR per your comment below.

Can we remove it?

Reply:
sure, deleted it now